python/packages/autogen-core/src/autogen_core/models/_types.py

```diff
@@ -51,6 +51,7 @@ class FunctionExecutionResult(BaseModel):
 
     content: str
     call_id: str
+    is_error: bool = False
```

Comment:
My last suggestion was bad. It should be None for default. Using False is misleading and can lead to caller believing there isn't error while this is not set.